### PR TITLE
[CIRC-556] In transit report: remaining call number fields

### PR DESCRIPTION
**Pull request by OleksandrVidinieiev**
## Purpose
Add call number components and copy number to items in transit report.

Resolves [CIRC-556](https://issues.folio.org/browse/CIRC-556).

### CODE REVIEW

File: src/test/java/api/requests/ItemsInTransitReportTests.java [hunk 1]
```diff
@@ -580,13 +587,12 @@ private ItemBuilder createSmallAngryPlanetItemBuilder() {
...
       materialTypesFixture.book().getId(),
       loanTypesFixture.canCirculate().getId(),
-      StringUtils.EMPTY,
-      "ItemPrefix",
-      "ItemSuffix",
-      "")
+      "55555",
+      "PREFIX",
+      "SUFFIX",
+      "Copy 1")
```
Comment (marcjohnson-kint) [inline src/test/java/api/requests/ItemsInTransitReportTests.java]: why are these parameters on separate lines?

File: ramls/items-in-transit.json [hunk 2]
```diff
@@ -210,6 +210,15 @@
...
             }
+          },
+          "copyNumber": {
+            "type": "string",
+            "description": "Copy number is the piece identifier. The copy number reflects if the library has a copy of a single-volume monograph; one copy of a multi-volume, (e.g. Copy 1, or C.7.)"
+          },
+          "effectiveCallNumberComponents": {
+            "description": "Elements of a full call number",
+            "$ref": "schema/call-number-components.json",
+            "readonly": true
```
Comment (Svitlana-Zmiivska) [inline ramls/items-in-transit.json]: Why does effectiveCallNumberComponents have readonly property but copyNumber hasn't?

File: ramls/items-in-transit.json [hunk 3]
```diff
@@ -210,6 +210,15 @@
                 }
               }
             }
+          },
+          "copyNumber": {
```
Comment (bohdan-suprun) [inline ramls/items-in-transit.json]: This property is deprecated. The `effectiveCallNumberComponents` already has `copyNumber` property.

Comment (bohdan-suprun) [reply ramls/items-in-transit.json]: Just ignore. Mixed it up with `callNumber`

File: src/test/java/api/requests/ItemsInTransitReportTests.java (hunk 1 above)
Comment (OleksandrVidinieiev) [reply src/test/java/api/requests/ItemsInTransitReportTests.java]: fixed

File: ramls/items-in-transit.json (hunk 2 above)
Comment (OleksandrVidinieiev) [reply ramls/items-in-transit.json]: removed

**Pull request merged**